### PR TITLE
Maps2

### PR DESCRIPTION
**Pull request by niru32868**
Add Map activity in the app. The problem remains: app crash when getting permission of the map.

### CODE REVIEW

File: LetsGoApp2/app/src/main/java/dachman/lucas/letsgoapp2/EventViewActivity.java [hunk 1]
```diff
@@ -1,10 +1,16 @@
 package dachman.lucas.letsgoapp2;
 
+import android.*;
```
Comment (LucasDachman) [inline LetsGoApp2/app/src/main/java/dachman/lucas/letsgoapp2/EventViewActivity.java]: Can we remove this unneeded import?

File: LetsGoApp2/app/src/main/java/dachman/lucas/letsgoapp2/EventViewActivity.java [hunk 2]
```diff
@@ -118,7 +125,45 @@ private void populateViews() {
...
         //TODO: Start map activity or fragment
-        Toast.makeText(getApplicationContext(), "Maps not yet implemented", Toast.LENGTH_LONG).show();
+        getPermission();
+            //    ActivityCompat#requestPermissions
+            // here to request the missing permissions, and then overriding
+            //   public void onRequestPermissionsResult(int requestCode, String[] permissions,
+            //                                          int[] grantResults)
+            // to handle the case where the user grants the permission. See the documentation
+            // for ActivityCompat#requestPermissions for more details.
+            Intent intent = new Intent(getApplicationContext(), MapActivity.class);
```
Comment (LucasDachman) [inline LetsGoApp2/app/src/main/java/dachman/lucas/letsgoapp2/EventViewActivity.java]: We need to find a way to check if the user has given permission to use their location before starting the map activity. If the activity starts without the location permission, the app will crash.

File: LetsGoApp2/app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (LucasDachman) [inline LetsGoApp2/app/src/main/res/layout/activity_main.xml]: I think this file can be deleted.

File: LetsGoApp2/app/src/main/res/menu/menu_main.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (LucasDachman) [inline LetsGoApp2/app/src/main/res/menu/menu_main.xml]: Can we rename this file from 'menu_main.xml' to 'menu_maps.xml' ?

File: LetsGoApp2/app/src/main/java/dachman/lucas/letsgoapp2/EventViewActivity.java (hunk 1 above)
Comment (niru32868) [inline LetsGoApp2/app/src/main/java/dachman/lucas/letsgoapp2/EventViewActivity.java]: All unneeded imports can be remove. Sorry I forgot to remove them before.

File: LetsGoApp2/app/src/main/java/dachman/lucas/letsgoapp2/EventViewActivity.java (hunk 2 above)
Comment (niru32868) [inline LetsGoApp2/app/src/main/java/dachman/lucas/letsgoapp2/EventViewActivity.java]: I think we can set a if condition statement (like we did yesterday) or just ask user when our app launched.

File: LetsGoApp2/app/src/main/res/menu/menu_main.xml (hunk 4 above)
Comment (niru32868) [inline LetsGoApp2/app/src/main/res/menu/menu_main.xml]: Sure. And if the file name change, we also need to change some codes which connect to the menu. I  will do it and update the files.

File: LetsGoApp2/app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (niru32868) [inline LetsGoApp2/app/src/main/res/layout/activity_main.xml]: Yeah. This file I just use for testing the map activity deemo. We can deleted it and use our app UI.

File: LetsGoApp2/app/src/main/res/menu/menu_main.xml (hunk 4 above)
Comment (LucasDachman) [inline LetsGoApp2/app/src/main/res/menu/menu_main.xml]: If you use the refactor option it will do it for you

**Pull request merged**